common/src/main/java/com/LubieKakao1212/opencu/OpenCUConfigCommon.java

```diff
@@ -95,7 +95,7 @@ public static class Defaults {
         public static final OpenCUConfigCommon.DispenserDeviceConfig VANILLA_DISPENSER_DEVICE = new DispenserDeviceConfig(90, 5.0, 5.0, 1.0, 500, 1.0);
         public static final OpenCUConfigCommon.DispenserDeviceConfig GOLDEN_DISPENSER_DEVICE = new DispenserDeviceConfig(180, 10.0, 360.0, 1.0, 300, 1.0);
         public static final OpenCUConfigCommon.DispenserDeviceConfig DIAMOND_DISPENSER_DEVICE = new DispenserDeviceConfig(120, 2.5, 360.0, 1.5, 750, 1.5);
-        public static final OpenCUConfigCommon.DispenserDeviceConfig NETHERITE_DISPENSER_DEVICE = new DispenserDeviceConfig(15, 1.5, 10.0, 2.0, 1500, 3.0);
+        public static final OpenCUConfigCommon.DispenserDeviceConfig NETHERITE_DISPENSER_DEVICE = new DispenserDeviceConfig(50, 0, 0.0, 5.5, 1500, 5.0);
```

Comment:
How much damage does this do?

Reply:
Not as much as you'd think, it's 1-1.5 hearts on a player wearing full Quantum + Enchanted (with instant dmg 2), and 2 shots an Iron Golem which I do recall Lubie wanted when we spoke.